packages/core/test/config.js

```diff
@@ -1,26 +1,25 @@
-var assert = require("chai").assert;
-var fs = require("fs-extra");
-var glob = require("glob");
-var { default: Box } = require("@truffle/box");
-var WorkflowCompile = require("@truffle/workflow-compile");
-var Ganache = require("ganache");
-var provision = require("@truffle/provisioner");
-var Resolver = require("@truffle/resolver");
-var Artifactor = require("@truffle/artifactor");
+const { assert } = require("chai");
+const WorkflowCompile = require("@truffle/workflow-compile");
+const Ganache = require("ganache");
+const provision = require("@truffle/provisioner");
+const Config = require("@truffle/config");
+const tmp = require("tmp");
+const path = require("path");
+const fse = require("fs-extra");
+let tempDir, config;
 
 describe("config", function () {
-  var config;
-  var customRPCConfig = {
+  const customRPCConfig = {
     gas: 90000,
     gasPrice: 2,
     from: "0x1234567890123456789012345678901234567890"
   };
 
-  before("Create a sandbox with extra config values", async function () {
+  before(async function () {
     this.timeout(5000);
-    config = await Box.sandbox("default");
-    config.resolver = new Resolver(config);
-    config.artifactor = new Artifactor(config.contracts_build_directory);
+    tempDir = tmp.dirSync({ unsafeCleanup: true });
+    fse.copySync(path.join(__dirname, "sources/metacoin"), tempDir.name);
+    config = new Config(undefined, tempDir.name);
```

Comment:
Yeah I thought the same but was calling the `removeCallback` manually in the tests which made creating a helper a bit messier. I figured out that `tmp` automatically cleans up after itself though so I think it will be cleaner to have this be a helper now.